ertviz/controllers/ensemble_selector_controller.py

```diff
@@ -100,3 +93,40 @@ def update_ensemble_selection(selected_nodes):
         }
 
         return data
+
+    @app.callback(
+        [
+            Output(parent.uuid("ensemble-selector"), "className"),
+            Output(parent.uuid("ensemble-selector-container"), "className"),
+            Output(parent.uuid("ensemble-selector-button"), "children"),
+        ],
+        [
+            Input(parent.uuid("ensemble-selector-button"), "n_clicks"),
```

Comment:
Could this be replaced by a checkbox instead? You wouldn't need to count the number of clicks

Reply:
The whole view changes place and size and label. I think that checkboxes makes more sense when it is standing still. n_clicks is the default click trigger for buttons.

Reply:
Ok, you made your case :)